include/seastar/net/arp.hh

```diff
@@ -47,7 +47,12 @@ public:
     arp_for_protocol(arp& a, uint16_t proto_num);
     virtual ~arp_for_protocol();
     virtual future<> received(packet p) = 0;
-    virtual bool forward(forward_hash& out_hash_data, packet& p, size_t off) { return false; }
+    virtual bool forward(forward_hash& out_hash_data, packet& p, size_t off) {
+      (void)out_hash_data;
```

Comment:
instead of using the good-old C-style cast, i'd suggest use `std::ignore`. just for better readability, so developer is aware that this value is intentionally *ignored*.

Reply:
TIL